modules/assim.batch/vignettes/MultiSitePDAVignette.Rmd

```diff
@@ -0,0 +1,646 @@
+---
+title: "HB-PDA vignette"
+author: "Istem Fer"
+date: "3/2/2018"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Multi-Site Hierarchical PDA
+
+This vignette documents the steps and settings of Hierarchical Bayesian Parameter Data Assimilation (HB-PDA) analysis on multiple sites. If you haven't done the standard PDA check that vignette first modules/assim.batch/vignettes/AssimBatchVignette.Rmd
+
+## Start with multi-settings
+
+Add the `<sitegroup>` tag in your `pecan.xml` for the groups of sites you are interested in. In this vignette, we will use HB-PDA site group in BETYdb which consists of the temperate decidious broadleaf forest (DBF) sites of Ameriflux network.
+
+```
+<sitegroup>
+   <id>1000000022</id>
+</sitegroup>
+```
+
+In the meantime, your `<run>` section should specify `<inputs>`, `<start.date>` and `<end.date>` information accordingly. E.g.:
+
+
+```
+ <run>
+  <settings.1>
```

Comment:
I didn't think you needed the .n at the end of settings. I've never used that before when setting up multisettings runs

Reply:
hmm, I can't remember whether I came up with this numbering or just mimicking what pecan workflow was automatically producing but it doesn't break anything and I thought it makes things easier to follow

Reply:
Ok, I just went back through earlier multisettings runs and it's something that was already there, it was just something that I'd never looked at before because my sitegroup runs expand to that automatically without me going and looking at them.

I agree the numbering is useful.

Reply:
IIRC the numbering doesn't serve any practical purpose, except to not have a bunch of identically named blocks. As in, I don't know what XML2list does if it finds a bunch of identically named list items at the same level in the hierarchy, but I think I just wanted to have control over that.